pkgs/applications/misc/blender/default.nix

```
@@ -13,10 +13,14 @@
 }:
 
 with lib;
+let
+  python = python3Packages.python;
+  optix = fetchzip {
+    url = "https://developer.download.nvidia.com/redist/optix/v7.0/OptiX-7.0.0-include.zip";
+    sha256 = "1b3ccd3197anya2bj3psxdrvrpfgiwva5zfv2xmyrl73nb2dvfr7";
+  };
 
-let python = python3Packages.python; in
-
-stdenv.mkDerivation rec {
+in stdenv.mkDerivation rec {
```

Minor suggestion, which reduces number of modified lines by 1

```suggestion
in

stdenv.mkDerivation rec {
```